types/cacheable-request/index.d.ts

```diff
@@ -19,8 +20,10 @@ declare const CacheableRequest: CacheableRequest;
 
 type RequestFn = typeof request;
 
+export type StorageAdapter = Store<any>;
```

Comment:
Better:
```ts
export {Store} from 'keyv';
```

Reply:
Ah, thanks for explaining that!

I moved it into the namespace, though I left the templated type in there to keep this streamlined, since as far as I can tell, it's not changeable by the caller.